railties/lib/rails/application/bootstrap.rb

```diff
@@ -55,7 +55,11 @@ module Bootstrap
           logger
         end
 
-        unless Rails.logger.is_a?(ActiveSupport::BroadcastLogger)
+        if Rails.logger.is_a?(ActiveSupport::BroadcastLogger)
+          if config.broadcast_log_level
```

Comment:
Is this a new config?

Reply:
It's an alias: https://github.com/rails/rails/pull/50337/files#diff-9f26e965e7e2a6842bd6ea755924a621038bc9dd8d63cfa6b34092aa32ede02cR376-R383

Reply:
Yep it's an alias ish, also added a `:nodoc:` as there is no reason to use is for the user.